src/js/config/Constants.js

```diff
@@ -1,5 +1,15 @@
-import { Platform } from 'react-native';
+import { Platform, Dimensions } from 'react-native';
 
-export const IS_IOS = Platform.OS === 'ios';
 export const TEST_PREFIX = 'test-';
 export const MAKE_ACCESSIBLE_FOR_AUTOMATION = { accessible: false };
+export const IS_IOS = Platform.OS === 'ios';
+export const {
+  width: WINDOW_WIDTH,
+  height: WINDOW_HEIGHT,
+} = Dimensions.get('window');
+export const IS_IPHONEX = IS_IOS &&
+  !Platform.isPad &&
+  !Platform.isTVOS &&
+  ((WINDOW_WIDTH === 812 || WINDOW_HEIGHT === 812) || (WINDOW_WIDTH === 896 || WINDOW_HEIGHT === 896));
+export const MUSEO_SANS_NORMAL = IS_IOS ? 'MuseoSans-500' : 'MuseoSans_500';
+export const MUSEO_SANS_BOLD = IS_IOS ? 'MuseoSans-900' : 'MuseoSans_900';
```

Comment:
This is somewhat surprising to me - I see that the filename in both /assets and /android is MuseoSans_900, is there a filename conversion happening in the iOS app on build?

Reply:
Fixed it by adding a comment